services/movie_session.py

```diff
@@ -0,0 +1,53 @@
+from typing import Optional
+from db.models import MovieSession
+from datetime import datetime
+
+
+def create_movie_session(
+    movie_show_time: int, movie_id: int, cinema_hall_id: int
+) -> None:
+    new_movie_session = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return new_movie_session
+
+
+def get_movies_sessions(
+    session_date: Optional[str] = None,
+) -> list[MovieSession]:
+    if session_date is not None:
+        date_obj = datetime.strptime(session_date, "%Y-%m-%d").date()
+        session = MovieSession.objects.filter(show_time__date=date_obj)
+        return session
+    return MovieSession.objects.all()
```

Comment:
```suggestion
    sessions = MovieSession.objects.all()
    if session_date is not None:
        date_obj = datetime.strptime(session_date, "%Y-%m-%d").date()
        sessions = sessions.filter(show_time__date=date_obj)
    return sessions
```
Better to do like that, it helps to support code in future